14/umbraco-cms/extending/packages/installing-and-uninstalling-packages.md

```diff
@@ -21,10 +21,6 @@ In the Umbraco Backoffice, you will find a **Packages** section that displays th
 
 ![Backoffice - Packages section](images/backoffice-packages-section.png)
 
-{% hint style="warning" %}
```

Comment:
Probably can be removed in the 13 docs as well.

Reply:
Makes sense! I have removed it for v13 and 12, but left it in v10 as I believe that will probably be the first version most people going from v8 will go to :)